src/components/Create.tsx

```diff
@@ -76,12 +77,13 @@ class Create extends React.Component<Props, State> {
     try {
       this.setState({ isSubmitting: true });
       const { name, m, n } = this.state;
-      await api.createWallet({
+      const wallet = await api.createWallet({
         name,
         m: parseInt(m, 10),
         n: parseInt(n, 10),
       });
-      this.props.getWallets();
+      await this.props.getWallets();
+      this.props.changeWallet(wallet);
```

Comment:
Maybe we should have a new action that's like `addWallet` to avoid the need to refetch them all after creation, but very not important.